client/app/bundles/shared/components/Tag.scss

```diff
@@ -0,0 +1,54 @@
+@import "base.scss";
+
+.tag
+{
+  position: absolute;
```

Comment:
Let's not use `position: absolute;` here, since we would then have to nest every instance of a tag under something with relative positioning! It also makes tags behave very wonky as you resize the viewport.

If you could make it so this component encapsulates text in a styled container without absolute positioning, that would be great!

The spacing within the tag can be achieved by using `padding`.